client/src/EventView.js

```diff
@@ -68,31 +86,32 @@ const EventView = ({ history, location }) => {
     }
     axios.get(`${process.env.REACT_APP_API_DOMAIN}/api/events/${eventId}`)
       .then(response => {
-      
+
         response.data.event.startDate = new Date(response.data.event.startDate)
         response.data.event.endDate = new Date(response.data.event.endDate)
 
         setEvent(response.data.event)
+        setCanceled(response.data.event.canceled);
         document.title = `${response.data.event.title} | CE News`
         const userId = getUserId()
         if(userId) {
```

Comment:
Si no te deja ver los eventos al estar logeado como admin, es por esta linea, ya la corregi en otra branch pero para que puedas hacer pruebas aqui tienes que poner:
```suggestion
        if(userId && !isUserAdmin()) {
```
Tambien recueda incluir el archivo de `TokenUtilities.js` en la parte de arriba del archivo.